### PR TITLE
Testnet curation

### PR DESCRIPTION
**Pull request by jessebmiller**
* Adds Ropsten network to curation market truffle config
* Abstracts Makefile for network specific deployments

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,39 +13,51 @@ squad-chess = packages/app-spec-squad-chess
...
 
+.PHONY: %-squad-chess-alpha-server
+%-squad-chess-alpha-server: build/metastore build/%-squad-sdk-js load-%-defs
+	cd $(squad-chess) && npm run build
+	cd $(squad-chess) && npx http-server
 
-.PHONY test:
-test: test-curation-market test-squad-games-web test-app-spec-roshambo
-test: test-metastore test-squad-chess
+load-%-defs:
```
Comment (EzraWeller) [inline Makefile]: Abstract the package that we're loading the defs for.

File: packages/curation-market/README.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+The Squad Curation Market is a part of the
+[Squad-SDK](/packages/squad-sdk/README.md) that lets users buy and
+sell tokens representing game elements, lets creators make profit from
+these sales, and use curation markets to discover games and game
+elements.  The Curation Market is built on the Ethereum blockchain and
+uses curved bonds to create token markets.
+
+# Deploying to public nets
+
+put your mnumonic in a file named `.secret` in the root of the `app`
```
Comment (EzraWeller) [inline packages/curation-market/README.md]: mnemonic

**Review by EzraWeller — APPROVED** [review]

**Comment by jessebmiller** [comment]
Figured out why CI keeps failing and we need to make some changes to how we are handling this.

Rather than having the migration script write out the contract addresses, we should use the contract json ABIs and what Truffle is already keeping track of in there.

This can probably get rid of all the {network}-curation-config.json files

**Comment by jessebmiller** [comment]
Find the network (and look it up in the contract json['networks'][networkId]) with `web3.version.getNetwork((err, networkId) => {})`

**Pull request merged**